### ISSUE TITLE
Notification in light mode could not have icon

### ISSUE THREAD

**Issue by GMartigny** [buefy/buefy#4059]
### Overview of the problem

**Buefy** version: 0.9.29 (I don't know if this impact buefy-next)
**Vuejs** version: 2.7.15
**OS/Browser**: Chrome (irrelevant)

### Description

The `has-icon` prop doesn't work with light variant of notifications.

### Steps to reproduce

```vue
<b-notification
  type="is-danger is-light"
  has-icon
>
  I don't have an icon :'(
</b-notification>
```

### Expected behavior

Notifications should have their icon (when asked) regardless of the light mode or not.

### Actual behavior

https://codepen.io/GMartigny/pen/abeJRaN

### Workaround

One could use `is-light` in the `class` prop instead of the `type`.

This render as expected, but this is not what's in the documentation.

```vue
<b-notification
  type="is-danger"
  class="is-light"
  has-icon
>
  I don't have an icon :'(
</b-notification>
```

**Comment by wesdevpro** [comment]
The type prop is working as expected. I dont believe we implemented functionality for parsing multiple types and combining them together. Use should use the type prop to define what icon you want then use classes or sass vars to further style your notification

**Comment by GMartigny** [comment]
As of today, in the documentation, the `type` prop is used with two values for "light" variant. This is true for [`Notifications`](https://buefy.org/documentation/notification#types), but also for [`Buttons`](https://buefy.org/documentation/button#types-and-states) for example.

If this is not the correct usage of `type`, it should be changed in the documentation.

**Comment by wesdevpro** [comment]
> As of today, in the documentation, the `type` prop is used with two values for "light" variant. This is true for [`Notifications`](https://buefy.org/documentation/notification#types), but also for [`Buttons`](https://buefy.org/documentation/button#types-and-states) for example.
> 
> 
> 
> If this is not the correct usage of `type`, it should be changed in the documentation.
> 
> 

You are correct! The style also seems to properly combine via the type prop. 
I over looked that the actual issue which is the fact that the icon doesn't show when you have combined types
